docs/integrations/prefect-aws/index.md

```diff
@@ -113,23 +123,20 @@ def secrets_manager_flow():
     aws_secret.delete_secret()
     return secret_data
 
-secrets_manager_flow()
-```
-
-### Using Prefect with AWS ECS
 
-`prefect_aws` allows you to use [AWS ECS](https://aws.amazon.com/ecs/) as infrastructure for your deployments. Using ECS for scheduled flow runs enables the dynamic provisioning of infrastructure for containers and unlocks greater scalability. This setup gives you all of the observation and orchestration benefits of Prefect, while also providing you the scalability of ECS.
-
-See the [ECS guide](/ecs_guide/) for a full walkthrough.
+if __name__ == "__main__":
+    secrets_manager_flow()
+```
 
-## Resources
+### Run flows on ECS
 
-Refer to the API documentation on the sidebar to explore all the capabilities of Prefect AWS!
+Run flows on [AWS Elastic Container Service (ECS)](https://aws.amazon.com/ecs/) to dynamically scale your infrastructure.
 
-For more tips on how to use blocks and tasks in Prefect integration libraries, check out the [docs](https://docs.prefect.io/integrations/usage/)!
+See the [ECS guide](/ecs_guide/) for a walkthrough of using ECS in a hybrid work pool.
 
-For more information about how to use Prefect, please refer to the [Prefect documentation](https://docs.prefect.io/).
+If you're using Prefect Cloud and your organization's security posture allows storing credentials in blocks, [ECS push work pools](https://docs.prefect.io/latest/guides/deployment/push-work-pools/#__tabbed_1_1) are a great option.
+They provide all the benefits of ECS with a quick setup and no worker needed.
 
-If you encounter any bugs while using `prefect-aws`, feel free to open an issue in the [`prefect`](https://github.com/PrefectHQ/prefect) repository.
+## Next steps
```

Comment:
I don't think we need this

Reply:
Changed to resources and moved a bit more, hoping to follow a similar format for other libraries, so trying to keep it flexible. PR now includes a few updates to prefect-dbt to try to align with these changes in prefect-aws.